tests/test_routes.py

```diff
@@ -0,0 +1,58 @@
+# from flask.wrappers import Response
+
+# ^ huh ?? why was this in my hello books code ? 
+
+def test_get_planet_with_no_records(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_planet(client, two_planets):
+    response = client.get("/planets/1")
```

Comment:
We could made this test a _little_ less brittle, though perhaps a little tougher to read, if we write `two_planets` to return the created models as well. We could then access the id of the model here, using the id the database gave the record. This can help us future-proof our tests (in case we change the number or order of records created, directly or indirectly).